config/initializers/eln_features.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
-# pack version
-pack_path = Webpacker.manifest.send(:data)&.fetch('application.js', nil)
-ENV['VERSION_ASSETS'] = pack_path && File.basename(pack_path)
+# TODO: clarify what this is and why it is used in the message api
```

Comment:
somehow that should notify the user to reload the page/assets if their client app is running an older version